.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Test collection
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  changelog:
+    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
```

Comment:
@justjais FYI https://github.com/ansible-community/ansible-test-gh-action is a thing that is now integrated into the default collection template @ https://github.com/ansible-collections/collection_template and will eventually be a part of ansible-core's suggested skeleton too.

Reply:
@webknjaz thanks so much for the information.